src/containers/guest-home-page/google-login/google-login.style.js

```diff
@@ -18,7 +18,7 @@ export const style = {
   google: {
     marginBottom: '16px',
     width: '100%',
-    padding: { xs: '15px 63.5px', sm: '15px 78px' }
+    padding: '15px 0'
```

Comment:
just py: '15px'

Reply:
@MickhaelSh Fix this, and you can merge.

Reply:
changed